dataactcore/read_config.py

```diff
@@ -0,0 +1,165 @@
+import os
+import os.path
+from os.path import expanduser, normpath, dirname, abspath
+import yaml
+import re
+
+VALUES_TO_LOG = {"CONFIG_BROKER":["path","use_aws","local",'aws_bucket', 'aws_role', 'aws_region', 'aws_create_temp_credentials',
```

Comment:
Can we potentially move this list itself into the config (or a not-to-log)? That way we won't have to modify config.py every time we add an item to config.yml. We'll just have to make a change to the config file we're already in.